Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -333,6 +335,50 @@ def determine_indicator_type(indicator_type, auto_detect, value):
     return indicator_type
 
 
+def merge_indicators_service(first_indicator, second_indicator):
```

Comment:
Add the type for the function signature and return type to annotation, e.g. 
def merge_indicators_service(first_indicator: Dict, second_indicator: Dict) -> : Dict: